lms/templates/admin/base.html.jinja2

```diff
@@ -72,6 +72,8 @@
                            href="{{ request.route_url("admin.registrations") }}">LTI Registrations</a>
                         <a class="navbar-item"
                            href="{{ request.route_url("admin.organizations") }}">Organizations</a>
+                        <a class="navbar-item"
+                           href="{{ request.route_url("admin.courses") }}">Courses</a>
```

Comment:
Added a top level item for now.

I reckon in the future it will make more sense to use a macro and make the search from both orgs and ais.